File: c++/ARRAY/Factorial.cpp
#include<iostream>
using namespace std;
int main()
{
    int num, i, fact=1;
    cout<<"enter a positive number:";
    cin>>num;
    if(num<0)
        cout<<"Error! factorial of negative number doesn't exist";
    else 
    {for(i=1; i<=num; i++)
        fact=fact*i;
    cout<<"Factorial is:"<<fact<<endl;
    }
    
    return 0;
}

File: java/array/Factorial.java
package array;
// import java.util.Scanner;
public class Factorial
{
    public static void main(String[] args)
    {
        int num=5, result=1;
        for(int i=num; i>0; i--)
        {
            result=result*i;
        }
        System.out.println("Result is: "+result);
    }
}


// public class Factorial
// {
//     public static void main(String[] args)
//     {
//         try (Scanner sc = new Scanner(System.in)) {
//             System.out.println("Enter number: ");
//             int num = sc.nextInt();
//             int fact=1;
//             for(int i=num; i>=1; i--)
//                 fact = fact*i;

//             System.out.println("Factorial is: "+fact);
//         }
//     }
// }